tutorials/strapi-app-serverless-containers-sqldb/index.mdx

```diff
@@ -0,0 +1,833 @@
+    ```bash
+    export SCW_APPLICATION_SECRET=$(scw iam api-key create application-id=$SCW_APPLICATION_ID -o json | jq -r '.secret_key')
+    ```
+
+4. Retrieve your container's ID:
+    ```bash
+    export CONTAINER_ID=$(scw container container list name="my-strapi-blog" -o json | jq -r '.[0].id')
+    ```
+
+8. Redeploy your Serverless Container:
+    ```bash
+    scw container container update $CONTAINER_ID \
+    environment-variables.DATABASE_HOST=$DATABASE_HOST \
+    environment-variables.DATABASE_PORT=$DATABASE_PORT \
+    environment-variables.DATABASE_NAME=$DATABASE_NAME \
+    environment-variables.DATABASE_CLIENT=$DATABASE_CLIENT \
+    environment-variables.DATABASE_SSL=$DATABASE_SSL \
+    environment-variables.DATABASE_USERNAME=$SCW_APPLICATION_ID \
+    secret-environment-variables.0.key='DATABASE_PASSWORD' \
+    secret-environment-variables.0.value=$SCW_APPLICATION_SECRET \
+    secret-environment-variables.1.key='APP_KEYS' \
+    secret-environment-variables.1.value=$APP_KEYS \
+    secret-environment-variables.2.key='API_TOKEN_SALT' \
+    secret-environment-variables.2.value=$API_TOKEN_SALT \
+    secret-environment-variables.3.key='ADMIN_JWT_SECRET' \
+    secret-environment-variables.3.value=$ADMIN_JWT_SECRET \
+    secret-environment-variables.4.key='TRANSFER_TOKEN_SALT' \
+    secret-environment-variables.4.value=$TRANSFER_TOKEN_SALT \
+    secret-environment-variables.5.key='JWT_SECRET' \
+    secret-environment-variables.5.value=$JWT_SECRET redeploy=true
+    ```
+
+9. Refresh your browser page displaying Strapi Administration Panel. An updated version displays. 
+
+  Congratulations, you have deployed a full serverless Strapi blog example!
+   
+## Going further with containers
+
+- Inspect your newly created resources in the Scaleway console:
+
+  - You can display your **Registry namespace** and **container image** in the [Container Registry section](https://console.scaleway.com/registry/namespaces).
+
+  - You can display your **Serverless Containers namespace** and **container deployment** in the [Serverless Containers section](https://console.scaleway.com/containers/namespaces).
+
+  - You can display your **Serverless SQL Database** in the [Serverless SQL Databases section](https://console.scaleway.com/serverless-db/databases).
+
+- **Fine-tune deployment options** such as autoscaling, targeted regions, and more. You can find more information by typing `scw container deploy --help` in your terminal, or by referring to the [dedicated documentation](/serverless/containers/how-to/manage-a-container/)
+
+- Create a secondary development environment by duplicating your built container, building it in `NODE_ENV=development` environment, running `npm run develop`, and plugging it onto another **Serverless SQL Database**. This will let you for instance edit content types which is not possible in production.
```

Comment:
```suggestion
- Create a secondary development environment by duplicating your built container, building it in `NODE_ENV=development` environment, running `npm run develop`, and plugging it onto another **Serverless SQL Database**. This will let you, for instance, edit content types, which is not possible in production.
```